src/main/java/com/thealgorithms/sorts/ExchangeSort.java

```diff
@@ -0,0 +1,39 @@
+package com.thealgorithms.sorts;
+
+import static com.thealgorithms.sorts.SortUtils.*;
+
+/**
+ * Exchange Sort (Cocktail Shaker Sort) implementation.
+ *
+ * @author 555vedant (Vedant Kasar)
+ * @see SortAlgorithm
+ */
```

Comment:
I have already provided link of documentation of exchange sort!!

Reply:
Exchange sort works by comparing the first element with all elements above it, swapping where needed, thereby guaranteeing that the first element is correct for the final sort order; it then proceeds to do the same for the second element, and so on. It lacks the advantage which bubble sort has of detecting in one pass if the list is already sorted, but it can be faster than bubble sort by a constant factor (one less pass over the data to be sorted; half as many total comparisons) in worst case situations.

Reply:
Is this fine?

Reply:
First of all: please put it into the code.

Regarding the text itself, it can be shortened. Please have a look at [stackoverflow](https://stackoverflow.com/questions/40786409/whats-the-difference-between-exchange-sort-and-bubble-sort-are-they-the-same).